yapapi/executor/__init__.py

```diff
@@ -227,12 +227,14 @@ async def process_invoices() -> None:
                     allocation = self._allocation_for_invoice(invoice)
                     try:
                         await invoice.accept(amount=invoice.amount, allocation=allocation)
-                    except Exception:
+                    except (Exception, CancelledError) as e:
                         emit(
                             events.PaymentFailed(
                                 agr_id=invoice.agreement_id, exc_info=sys.exc_info()  # type: ignore
                             )
                         )
+                        if isinstance(e, CancelledError):
+                            raise
```

Comment:
Do we want to display "payment failed" when the coroutine was cancelled? Maybe a separate `except` section with a different message would be a better idea?